libs/plugins/stream-client/src/lib/core/state/clean-dangling-tasks-output-mappings.ts

```diff
@@ -52,7 +49,7 @@ function cleanTasks(
         taskId,
         {
           ...task,
-          inputMappings: pick(task.inputMappings, outputNames),
+          output: pick(task.output, outputNames),
```

Comment:
this will also remove other output mappings that are not pipeline mappings, for example for the following set of mappings `otherOutput` will always be removed.

```
{
  "pipeline.x": "=foo",
  "pipeline.y": "=foo",
  "otherOutput": "=baz"
}
```